gradle/libs.versions.toml

```diff
@@ -40,7 +40,7 @@ ktor = "1.6.8"
 managed-logback = "1.4.5"
 logbook-netty = "2.14.0"
 log4j = "2.19.0"
-micronaut-aws = "3.9.2"
+micronaut-aws = "3.10.0"
```

Comment:
@melix why is the micronaut-aws still in core 4.0.x? Can we remove it?

Reply:
There are a few dependencies like `micronaut-aws` which are used for testing: https://github.com/micronaut-projects/micronaut-core/blob/5573f6a798d583462d997f163b1da016509d2f80/test-suite-kotlin/build.gradle#L73

and https://github.com/micronaut-projects/micronaut-core/blob/61d9ba73a3fbfdacf9038e57e4ad8658bbad4004/gradle/libs.versions.toml#L83-L85

Removing would imply refactoring the tests so that they don't use those libraries.